### ISSUE TITLE
Ldap is incorrect.  Both Users and Groups are mapped to Users.  Please fix.

### ISSUE THREAD

**Issue by fw100** [zentyal/zentyal#2078]
**Describe the bug**
LDAP  maps groups to users instead of groups.
**To Reproduce**

**Expected behavior**
Default Groups DN
CN=Groups,DC=anywhere,DC=local

**Screenshots**
 LDAP information
Base DN
DC=anywhere,DC=local
Default Users DN
CN=Users,DC=anywhere,DC=local
Default Groups DN
CN=Users,DC=anywhere,DC=local 

**Zentyal OS (please complete the following information):**
 - Version: [e.g. 5.0, 5.1, 6.0, 6.1]
 - Version of the module which has the bug [e.g. zentyal-dns-5.0.3]
 - Modules installed [e.g. DNS, DC, Jabber]
Community Edition 7.0

**Other OS (please complete the following information):**
 - OS: [e.g. Windows 10]
 - Version: [e.g. build 18363]
Ubuntu 20.04 LTC
**Additional context**

Can this be fixed by editing a file on the local computer?  I seriously messed up my Windows domain by syncing with Zentyal.

**Comment by Neustradamus** [comment]
@djoven89: Any news?

**Comment by fw100** [comment]
No, I haven’t heard anything.  I don’t think much time is spent on the community version.

 

Frank

**Comment by brunolorente** [comment]
Hi, @Neustradamus @fw100 I'm back, we cannot reproduce the issue right now.
This feature is stable, maybe I misunderstood the issue, could you please edit this issue following the template?

Regards!

**Comment by fw100** [comment]
I stopped working with Zentyal because there are too many unresolved errors.  You can close any open tickets.Thank you,FrankFrank M. Whitman

**Comment by stephen-amaero** [comment]
This is still an issue in 7.0
It also causes the Domain Admins Group to be listed as a User in the User Management screen
- I installed Zentyal 7.0 twice to see if it was something I was doing ... but it seems the project is broken.
![screenshot 95](https://user-images.githubusercontent.com/118324678/202069180-8dcf1716-58b9-4703-9470-bcdc8b4435a3.png)
![screenshot 96](https://user-images.githubusercontent.com/118324678/202069844-d45be78e-fd34-4f6d-bce3-7d9f7923ba0f.png)
